css/menu-styles.css

```diff
@@ -0,0 +1,64 @@
+.paw-logo {
+  display: inline-block;
+}
+.menu-grid {
+  max-width: 1072px;
+  margin: 200px auto 100px;
+  background-color: #000000;
+  border-radius: 20px;
+  border: 12px solid #ffc0cb;
+  box-sizing: border-box;
+  padding: 60px;
```

Comment:
this padding affects the heading adding the top padding that is not needed (this is an open discussion! I may be wrong. please share your thoughts)

Reply:
I made necessary changes in order it looked as close to the layout as I could. as far as I understood, this font Italianno has some default styles taken from user agent stylesheet, it took me almost 2 hours to manage it as it is now and still it's not perfect and it's margin-top doesn't correspond to pixels in figma. maybe I did something wrong and it's solution is very simple, but I can't do nothing with that)